src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import TodoList from './TodoList';
+
+const Todo = props => {
+    return (
+    <div>
+        {props.list.map(listsInMap => (
```

Comment:
i like your semantic naming of your cb parameter here gj thinking about what the data your passing around is